src/libs/antares/study/parts/short-term-storage/series.cpp

```diff
@@ -38,6 +40,44 @@ bool Series::validate() const
 
 bool Series::loadFromFolder(const std::string& folder)
 {
+    bool ret = true;
+
+    ret = loadFile(folder, "PMAX-injection.txt", maxInjection) && ret;
+    ret = loadFile(folder, "PMAX-withdrawal.txt", maxWithdrawal) && ret;
+    ret = loadFile(folder, "inflow.txt", inflows) && ret;
+    ret = loadFile(folder, "lower-rule-curve.txt", lowerRuleCurve) && ret;
+    ret = loadFile(folder, "upper-rule-curve.txt", upperRuleCurve) && ret;
+
+    return ret;
+}
+
+bool Series::loadFile(const std::string& folder, const std::string& filename, std::vector<double>& vect)
```

Comment:
1. Move to libs/antares, name this function something "loadSingleColumnFile"
2. Add 8760 as a parameter
3. Clear the vector before writing to it